1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,33 @@
-function setAlarm() {}
+function setAlarm() {
+  let inputValue = document.getElementById('alarmSet')
+  let remainingTime = document.getElementById('timeRemaining')
+
+  let minutes = ''
+  let seconds = ''
+
+ let interval = setInterval(()=>{
+  inputValue.value--
```

Comment:
I think the inputValue should not change in the UI. Only the remaining time should.

In order to do this you could store the inputValue.value outside the interval as in:


```


let value = inputValue.value--
 let interval = setInterval(()=>{
  value--
...


```
and change all the references to inputValue to the new variable

Reply:
thank you for this advise, working as it should